yt_dlp/extractor/zattoo.py

```diff
@@ -327,139 +319,547 @@ class ZattooRecordingsIE(ZattooBaseIE):
     }]
 
 
-class NetPlusIE(ZattooPlatformBaseIE):
-    _NETRC_MACHINE = 'netplus'
+class NetPlusTVBaseIE(ZattooPlatformBaseIE):
+    _NETRC_MACHINE = 'netplustv'
```

Comment:
Don't change netrc of existing extractor
```suggestion
    _NETRC = 'netplus'
```

Reply:
It used to be `_NETRC_MACHINE = 'netplus'`, not `_NETRC = 'netplus'`, I think this causes the tests to fail...

Reply:
Yeah, my mistake. I'll fix it